app/org/sagebionetworks/bridge/models/healthdata/HealthDataUserConsent.java

```diff
@@ -0,0 +1,85 @@
+package org.sagebionetworks.bridge.models.healthdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.joda.time.DateTime;
+
+import org.sagebionetworks.bridge.json.DateTimeToStringSerializer;
+import org.sagebionetworks.bridge.json.JodaDateTimeDeserializer;
+
+/**
+ * User consent metadata stored with the health data record. We store this in the Health Data Record table, so we can
+ * avoid doing a bunch of extra calls to other tables during export.
+ */
+@JsonDeserialize(builder = HealthDataUserConsent.Builder.class)
+public class HealthDataUserConsent {
```

Comment:
For the purposes of export functionality, that's true. For the purposes of actual research data, Andrew asked me for the consent date (strictly speaking, the join date, which is the same thing). And based on our conversation yesterday, there might be an interesting correlation between data sharing prefs and data. So at this point, I figured I might as well include all consent information.

Re: sharing scope - Yes, I'll need to integrate that. I started the TranscribeConsentHandler before the sharing scope stuff was finalized, and I didn't want to block on that before getting eyeballs on this code review. I'll definitely be sure to pull it in.